latex.ddoc

```diff
@@ -85,7 +85,6 @@ _=
 
 FOOTER=
 FOOTNOTE=\footnote{$0}
-FULL_XREF=\href{phobos/std_$1.html#$2}{$(D std.$1.$2)}
```

Comment:
`dlang.org.ddoc` defines `FULL_XREF=$(XREF $1, $2)` (i.e. `FULL_XREF` is an alias for `XREF`). The language reference only ever uses `FULL_XREF`, and hell would break loose in LaTeX if it used `XREF`. `FULL_XREF` seems to be a historical artifact.

By redefining `XREF` instead of `FULL_XREF` in `latex.ddoc`, they become synonymous there, too. Hell won't break loose if someone uses `XREF` instead of `FULL_XREF` in the language reference. And the `XREF` related definitions are next to each other in `latex.ddoc`.